airbyte-webapp/src/packages/cloud/cloudRoutes.tsx

```diff
@@ -120,6 +120,7 @@ const MainViewRoutes = () => {
             path={`${r}/*`}
             element={
               <Navigate
+                // @ts-expect-error state is now unkown, needs proper typing
```

Comment:
:information_source: `location.state` is now `unknown`. I've properly typed it in one file, where I could encapsulate custom type guards only within this file. This place here might need some utility type guard somewhere else, so left it untyped for now.